src/util/misc.ts

```diff
@@ -6,27 +6,34 @@ export { fast_deep_equal as deepEqual } from "../../deps.ts";
 
 export const deepCopy = rfdc();
 
-export let microsecondNow = () => Date.now() * 1000;
+export function microsecondNow() {
+    return Date.now() * 1000;
+}
 
-export let sleep = (ms: number) =>
-    new Promise((res, rej) => {
+export function sleep(ms: number) {
```

Comment:
`sleep` deserves a JSDoc since it's commonly used in the code and examples

Reply:
Cool, added.